src/delivery/cli/setup.rs

```diff
@@ -57,20 +59,25 @@ impl<'n> SetupClapOptions<'n> {
             path: value_of(&matches, "config-path"),
             pipeline: value_of(&matches, "pipeline"),
             project: value_of(&matches, "project"),
+            a2_mode: matches.is_present("a2-mode"),
         }
     }
 }
 
 impl<'n> Options for SetupClapOptions<'n> {
     fn merge_options_and_config(&self, config: Config) -> DeliveryResult<Config> {
-        let new_config = config
+        let mut new_config = config
             .set_server(&self.server)
             .set_user(&self.user)
             .set_enterprise(&self.ent)
             .set_organization(&self.org)
             .set_pipeline(&self.pipeline)
-            .set_project(&self.project);
-
+            .set_project(&self.project)
+            .set_a2_mode(self.a2_mode);
+        // A2 mode requires SAML right now
```

Comment:
So we can't use/login with local users in A2?

Reply:
@afiune no, A2 doesn't expose an API that we can authenticate against with credentials yet. See the discussion of state in the comments of SHACK-393 on JIRA.

Reply:
Understood! 👍